addons/core/translations/resources/en-us.yaml

```diff
@@ -271,11 +271,14 @@ credential-store:
   title: Credential Store
   title_plural: Credential Stores
   description: A credential stores is a collection of credentials and credential libraries.
+  actions:
+    create: New Credential Library
```

Comment:
I don't think this action fits within credential store. How about moving this into `credential-library.actions.create`? Unless the text should read 'New Credential Store' instead.

Reply:
100%, sorry I miss that one because it literally says "New Credential Library" and not "Store". Thanks!!